container/sagemaker/serve.py

```diff
@@ -314,7 +318,7 @@ def _wait_for_gunicorn(self):
     @contextmanager
     def _timeout(self, seconds):
         def _raise_timeout_error(signum, frame):
-            raise TimeoutError('timed our after {} seconds'.format(seconds))
+            raise Exception('timed our after {} seconds'.format(seconds))
```

Comment:
A- Why is this an Exception
B - timed our => timed out *

Reply:
Updated, raising TimeoutError

Reply:
[Very Minor] I think we should stick with "Timed out".

Reply:
Updated.